ovos_plugin_manager/coreference.py

```diff
@@ -107,18 +107,9 @@ def create(config=None):
 
 
 def get_coref_config(config=None):
+    from ovos_plugin_manager.utils.config import get_plugin_config
     config = config or Configuration()
-    lang = config.get("lang")
     if "intentBox" in config and "coref" not in config:
-        config = config["intentBox"] or {}
-        lang = config.get("lang") or lang
-    if "coref" in config:
-        config = config["coref"]
-        lang = config.get("lang") or lang
-    config["lang"] = lang or "en-us"
-    coref_module = config.get('module') or 'dummy'
-    coref_config = config.get(coref_module, {})
-    coref_config["module"] = coref_module
-    return coref_config
-
-
+        return get_plugin_config(config, "intentBox")
```

Comment:
this check is not needed, can return ` get_plugin_config(config, "intentBox") or  get_plugin_config(config, "coref")` directly

Reply:
if `intentBox` is not in config, current behavior would still return a config with `lang`. We could probably return `dict()` in the case of: `section not in config and module not in config`?